### PR TITLE
adds materialize support

### PR DESCRIPTION
**Pull request by cc3c**
Hello.

I've added basic [materialize](http://materializecss.com) support.

I'm working in another feature (with options) in order to fulfill my project's needs that I think can be useful: adding support for icons instead of text in prev / next links

Carles.

### CODE REVIEW

**Comment by mgwidmann** [comment]
Great, thanks!

You should be able to use icons already like `previous: render("prev_link.html", []), next: render("next_link.html", [])` where those functions are other phoenix EEX templates. If that doesn't work, let me know! Also, if you'd like to add that as an example (if its not already) to the README, I'll gladly accept a PR for that.

File: lib/scrivener/html.ex [hunk 1]
```diff
@@ -233,11 +242,17 @@ defmodule Scrivener.HTML do
   defp li_classes_for_style(paginator, page_number, :semantic) do
     if(paginator.page_number == page_number, do: ["active", "item"], else: ["item"])
   end
+  defp li_classes_for_style(_paginator, :ellipsis, :materialize), do: []
+  defp li_classes_for_style(paginator, page_number, :materialize) do
+    if(paginator.page_number == page_number, do: ["active"], else: ["waves-effect"])
```
Comment (mgwidmann) [inline lib/scrivener/html.ex]: You may want to make it configurable to be able to change the wave effect color if you're going to include it...

Comment (cc3c) [inline lib/scrivener/html.ex]: Well, I'd like, but I'm not sure what's the best way to change it as a general (not only materialize) solution to add additional classes to elements.

I've tested your solution about icons, it works perfectly and it's very cool, thanks. I've added an example to the README

**Comment by mgwidmann** [comment]
Awesome, this looks really good! Give me a little bit and I'll have it out in a release.

**Comment by mgwidmann** [comment]
`v1.5.0` released!

**Pull request merged**